JavaScript/JavaScriptSDK.Tests/Selenium/Tests.html

```diff
@@ -7,13 +7,28 @@
     <link rel="stylesheet" href="http://code.jquery.com/qunit/qunit-1.18.0.css">
     <script src="http://code.jquery.com/qunit/qunit-1.18.0.js"></script>
     <script src="http://sinonjs.org/releases/sinon-1.17.2.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/require.js/2.2.0/require.js"></script>
     <script type="text/javascript">
         window.AIResults = [];
         QUnit.testDone(function(result) {
             window.AIResults.push(result);
         });
     </script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/require.js/2.2.0/require.js"></script>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/10049593/15684731/3f249148-271d-11e6-89ed-d8d525b624f1.png)

That's what github shows me :)